planner/plansdk/plansdk.go

```diff
@@ -113,6 +114,10 @@ func (p *Plan) WithError(err error) *Plan {
 	return p
 }
 
+// MergePlans merges multiple Plans into one. The merged plan's packages, definitions,
+// and overlays is the union of the packages, definitions, and overlays of the input plans,
+// respectively. The install/build/start stages of the merged plans are taken from the _first_
+// buildable plan (order matters!). If no plan is buildable, returns a non-buildable plan.
```

Comment:
`If no plan is buildable, returns a non-buildable plan.` Should be `If no plan is buildable, returns an empty plan.`

Reply:
Well.. no. It returns a plan with dependencies, but with no "stages". And "no stages" == "not buildable"...

right?